katalog/kube-proxy-exporter/deploy.yml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2020 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-proxy-exporter
+  labels:
+    k8s-app: kube-proxy-exporter
+    k8s-proxied-app: kube-proxy
+spec:
+  selector:
+    matchLabels:
+      k8s-app: kube-proxy-exporter
+      k8s-proxied-app: kube-proxy
+  template:
+    metadata:
+      labels:
+        k8s-app: kube-proxy-exporter
+        k8s-proxied-app: kube-proxy
+    spec:
+      tolerations:
+      - operator: Exists
+      serviceAccountName: kube-proxy-exporter
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
+      hostNetwork: true
```

Comment:
Why do we need `hostNetwork: true` and `hostPID: true` to expose kube-proxy metrics?

Reply:
We need `hostNetwork` (not sure if we need hostPID) to proxy kube-proxy metrics. As we are not deploying it as a sidecar container we need to access at the host level.

I'll try to test it without `hostPID`

Reply:
You're totally right about `hostNetwork` but I don't think `hostPID` should be necessary.

Reply:
Tested without hostPID and it continue working. Removed it :)